test/articles.test.js

```diff
@@ -33,6 +33,31 @@ describe('/articles Route', () => {
         })
     })
 
+    describe('GET /articles/:id', () => {
+        it('should respond with single article of same id', (done) => {
+            const id = '5846399f9ba87af2501fb035'
+            chai.request(app)
+                .get(`/articles/${id}`)
+                .end((err, res) => {
+                    checkSingleArticleProperties(res.body)
+                    res.body._id.should.equal(id)
+                    done()
+                })
+        })
+
+        it('should respond with an error if an article id is not found', (done) => {
+            const id = '5846399f9ba87af2501fb03f'
+            chai.request(app)
+                .get(`/articles/${id}`)
+                .end((err, res) => {
+                    res.body.should.have.property('err')
+                    res.body.reqParams.id.should.equal(id)
```

Comment:
I think you should add ` res.body.should.have.property('reqParams')` to be consistant